src/main/resources/templates/index.html

```diff
@@ -45,31 +52,34 @@
     </form>
 
 </main>
-<div th:replace="include/footer :: footer"></div>
-
-<script>
-    var attempt = 0;
-
-    function createFbPins(row) {
-        var correctColors = $("#getColors").val();
-        var correctPositions = $("#getPositions").val();
-        console.log(correctColors,correctPositions)
-
-        var FbContainer = document.getElementById('feedback-container-${row}');
-        for(var i = 0; i< correctColors; i++) {
-            var pin = document.createElement('div');
-            FbContainer.appendChild(pin);
-            pin.classList.add('feedback-pin');
-
+    <div th:replace="include/footer :: footer"></div>
+
+    <script>
+        /*console.log(document.getElementsByClassName('feedback-container'))
```

Comment:
Die Zeile 58 bis 82 können gelöscht werden

Reply:
kann ich nicht löschen, ich brauche das für die Pins.

Reply:
Das kann man dann aber kopieren und in den neuen Branche einfügen

Reply:
okay